packages/headless/src/modules/order/services/customer.service.ts

```diff
@@ -38,8 +38,8 @@ export class OrderCustomerService {
   }
 
   async reOrder(userId: string, body: ReOrderQuery): Promise<ReOrderResponse> {
-    const { ignoreInvalidItems, overWriteCart, orderId } = body;
-
+    let { ignoreInvalidItems, overWriteCart, orderId } = body;
```

Comment:
Replace the line like
`const { ignoreInvalidItems=false, overWriteCart=false, orderId } = body;`
then you can check ===false or ===true without any issue.